--- ./install_original	2020-01-25 16:34:52.050000000 +0000
+++ ./install_fixed	2020-01-26 12:42:41.125955300 +0000
@@ -3,6 +3,10 @@

 (complete 0)

+; ADDED BY DUCKBENCH
+(set TRUE 1)
+(set FALSE 0)
+
 ;=============================================================================
 ; English strings

@@ -3387,7 +3391,7 @@
 ;=============================================================================
 ; say hi!

-(user 0)
+; REMOVED BY DUCKBENCH (user 0)
 (welcome)
 (set old_level @user-level)
