--- install	2020-06-26 22:27:38.930917500 +0100
+++ install2	2020-06-26 23:28:15.971070200 +0100
@@ -2,6 +2,8 @@
 ; Script to install Release 3.1 Workbench
 
 (complete 0)
+(set TRUE 1)
+(set FALSE 0)
 
 ;=============================================================================
 ; English strings
@@ -3438,7 +3440,7 @@
 ;=============================================================================
 ; say hi!
 
-(user 0)
+; REMOVED BY DUCKBENCH (user 0)
 (welcome)
 (set old_level @user-level)
 
@@ -3613,7 +3615,7 @@
 
 (if (exists cli-startup)
 (
-    (set oldfiles (cat oldfiles s/cli-startup "         \n"))   ; these are hard spaces
+    (set oldfiles (cat oldfiles cli-startup "         \n"))   ; these are hard spaces - FIXED BY DUCKBENCH
 ))
 
 (if (exists cd0)
@@ -4516,8 +4518,8 @@
 
 (user 2)
 (message #reboot)
-(run "Reboot")
-
+; (run "Reboot") NO REBOOT FOR DUCKBENCH
+ 
 )
 (
 
@@ -4572,6 +4574,6 @@
 ; "Say goodnight Gracy."
 ;
 (message #reboot-lang)
-(run "Reboot")
+;REMOVED BY DUCKBENCH (run "Reboot")
 
 ))
