--- install	2020-06-12 00:24:53.911867400 +0100
+++ install_patched	2020-06-12 00:25:46.537063000 +0100
@@ -2425,10 +2425,11 @@
 ;=============================================================================
 ; make sure we booted from the 2.1 Install disk

-(if (<> (getassign "SYS" "a") "Install2.1:")
-(
-    (abort #bad-boot)
-))
+; REMOVED BY DUCKBENCH
+;(if (<> (getassign "SYS" "a") "Install2.1:")
+;(
+;    (abort #bad-boot)
+;))


 ;=============================================================================
