--- install	2020-06-27 15:22:50.432164700 +0100
+++ install2	2020-06-27 15:24:39.126291400 +0100
@@ -3519,7 +3519,7 @@
 
       (user 2)
       (message #prereboot)
-      (reboot)
+      ; (reboot) NO REBOOT IN DUCKBENCH
       )
 
       (
@@ -4638,7 +4638,7 @@
       (user 2)
       (message #reboot)
       (closemedia instpic4)
-      (reboot)
+      ; (reboot) NO REBOOT IN DUCKBENCH
       )
       (
       (complete 100)
